src/shared/utils.js

```diff
@@ -163,4 +162,11 @@ export function loadFromFile(file) {
 
     return null;
   });
+
+  return {
+    promise,
+    abort: () => {
+      reader.abort();
+    },
+  };
```

Comment:
I'm wondering, if perhaps making the API consistent with [make-cancelable-promise](https://www.npmjs.com/package/make-cancellable-promise) by changing `abort` to `cancel` wouldn't be a bettter choice.

Reply:
Both ok for me. I am thinking: 'cancel' is something like "I did the request, but I don't want it anymore, don't send what ever you have, no matter if finished or not" (which is what cancelable-promise does). "Abort" on the other hand is "Stop what you are doing right now". But from our perspective I guess what we want indeed is the same in both cases: don't call any callbacks. So 'cancel' would make sense.

Reply:
changed it in an extra commit